configure

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/bash
-
+MAGE_KEY=
```

Comment:
This is needed due to `set nounset` below making the script fail if there is no definition of `MAGE_KEY`

Reply:
I'm happy to remove it, WDYT @nuclearsandwich since that has been there since the first creation of the script.

Reply:
These scripts get enough attention that I don't think we need it. If we get the POSIX sh PRs in I think it will make even more sense to remove since a lot of the techniques for mitigating nounset usage are Bash-specific or harder in POSIX sh.